internal/golada/builder/builder.go

```diff
@@ -165,6 +165,10 @@ func (b Builder) BuildFile() (by []byte, err error) {
 			return nil, e
 		}
 
+		for _, file := range topLevelDir.Files() {
```

Comment:
This line here adds files that are in the top-level directory to the debug output